bin/sl-runctl.js

```diff
@@ -189,14 +189,14 @@ function requestHeapSnapshot() {
   var name = optionalArg(util.format('node.%s', target));
   request.cmd = 'heap-snapshot';
   request.target = target;
-  // .heapdump extention Required by Chrome
-  request.filePath = path.resolve(name + '.heapdump');
+  // .heapdump extention required by Chrome
```

Comment:
comment is wrong, just loaded .heapsnapshot into chrome. pmctl has been .snapshot for a fair while.